### PR TITLE
March Update

### PR DESCRIPTION
**Pull request by garrying**
- [x] Updated reflow for typing text in paragraph on mobile (remove line-break on mobile)
- [x] Added extra time between typing changes (+1.5s)
- [x] Vertical spacing consistency for buttons
- [x] Follow-up on scrollbar styling (b54d768)
 in https://github.com/ournetworks/ournetworks.ca/pull/91#issuecomment-469023991
  - @dcwalk do you have a screenshot of what you're seeing?
- [x] Add in support/donate page (https://github.com/ournetworks/2019/issues/11 ->> https://github.com/ournetworks/ournetworks.ca/pull/93/commits/97fc85c0858bb1194f8f7b680f599dd7f58cc29b)

### CODE REVIEW

**Comment by dcwalk** [comment]
@garrying I think it is default behaviour, and I know firefox scrollbar styling is not a thing... but looks a little odd to have a box around the email typing section...and then a larger scrollbar for the page. Maybe just to me?

scrollbar

**Comment by garrying** [comment]
@dcwalk I've added new scrollbar properties in b54d768, referencing https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Scrollbars. Patch is for Firefox v64 and up.

**Comment by dcwalk** [comment]
Can we tack on the support page to this PR? That would close out https://github.com/ournetworks/2019/issues/11

**Review by dcwalk — APPROVED** [review]
I dig everything you've added @garrying -- tested and the potentially more subtle scrollbar is nice!

**Comment by dcwalk** [comment]
@garrying in a horrible bit of role reversal, can you review https://github.com/ournetworks/ournetworks.ca/pull/93/commits/97fc85c0858bb1194f8f7b680f599dd7f58cc29b? :pray: 

I think we can get the go-ahead and discuss on Sat, weekend merge maybe?!

**Comment by dcwalk** [comment]
Okay added link to paypal page! Should we have a toast/whatever when they come back to the page from the donate (maybe just on success?)?

**Comment by garrying** [comment]
I think just on success is fine. Will it look something like `/support-us?donation=confirmed`? We could reuse the signup JS bit.

**Comment by garrying** [comment]
pins-stickers
tote-design

Adding these swag shots to page with dithering inspo from lowtechmagazine.com

**Comment by dcwalk** [comment]
Dithering looks great! 

Only thing about placement: the button and any mention of donating is pushed below the fold, which seems a little ambiguous given the page name. Can we move the images even below the first sentence? 

Testing two options:

## OPTION 1
option1

## OPTION 2
option2

File: support-us.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  <div class="six columns">
+    <figure>
+      <img src="/images/swag/pins-stickers.png" alt="Buttons and stickers from 2018 Our Networks">
+      <figcaption><p>Our Networks buttons and stickers.</p></figcaption>
+    </figure>
+  </div>
+  <div class="six columns">
+    <figure>
+      <img src="/images/swag/tote-design.png" alt="Front of 2018 Our Networks Tote">
+      <figcaption><p>Printed front and back with shoutouts to our wonderful sponsors.</p></figcaption>
```
Comment (dcwalk) [inline support-us.md]:
```suggestion
      <figcaption><p>Totes printed front and back with shoutouts to our wonderful sponsors.</p></figcaption>
```

Comment (dcwalk) [reply support-us.md]: this agrees with how we refer to other swag (plural--buttons and stickers) and makes clear it is a tote!

**Comment by dcwalk** [comment]
Aaaand added in the notification on success!

**Comment by garrying** [comment]
Thinking option 2 makes sense. Could we do a re-review and merge?

**Review by dcwalk — COMMENTED** [review]
Looks good!

I think we are good to go, only potential change is to tone down the language (my fault)

File: support-us.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  <div class="six columns">
+    <figure>
+      <img src="/images/swag/pins-stickers.png" alt="Buttons and stickers from 2018 Our Networks">
+      <figcaption><p>Our Networks buttons and stickers.</p></figcaption>
+    </figure>
+  </div>
+  <div class="six columns">
+    <figure>
+      <img src="/images/swag/tote-design.png" alt="Front of 2018 Our Networks Tote">
+      <figcaption><p>Totes printed front and back with shoutouts to our wonderful sponsors.</p></figcaption>
```
Comment (dcwalk) [inline support-us.md]:
```suggestion
      <figcaption><p>Totes printed front and back with shoutouts to our sponsors.</p></figcaption>
```

**Review by dcwalk — APPROVED** [review]

**Pull request merged**